pylib/mozhginfo/mozhginfo/pushlog_client.py

```diff
@@ -29,16 +31,16 @@ class PushlogError(Exception):
     pass
 
 
-def query_revisions_range(repo_url, from_revision, to_revision, version=2, tipsonly=1):
+def query_pushes_by_revision_range(repo_url, from_revision, to_revision, version=2, tipsonly=1):
     """
-    Return an ordered list of revisions (by date - oldest (starting) first).
+    Return an ordered list of pushes (by date - oldest (starting) first).
 
-    repo           - represents the URL to clone a repo
+    repo_url           - represents the URL to clone a repo
     from_revision - from which revision to start with (oldest)
     to_revision   - from which revision to end with (newest)
     version        - version of json-pushes to use (see docs)
```

Comment:
Can you please align the "-" symbol for these lines to be vertically properly aligned? Thanks!